config/sync/migrate_plus.migration.d7_file.yml

```diff
@@ -17,7 +15,7 @@ source:
   scheme: public
   batch_size: 1000
   constants:
-    source_base_path: /app/imports/files
+    source_base_path: 'https://www.northernireland.gov.uk/sites/default/files/'
```

Comment:
One currently active D7 hostname. In fact, it might be prudent to always check this is a D7 site in future otherwise you might end up trying to copy files from yourself... depending on how DNS/domains are set up within platform.